### PR TITLE
Add skygear config to initialize plugins

### PR DESCRIPTION
**Pull request by ben181231**
connect #70

### CODE REVIEW

**Comment by oursky-travis** [comment]
@ben181231, thanks for your PR! By analyzing the annotation information on this pull request, we identified @rickmak to be a potential reviewer

**Comment by ben181231** [comment]
Renaming the original `config` to `skyconfig` is necessary since `py-skygear` load `.config` before loading user's cloud code. So, when `skygear.config('plugin_name')` is used in user's cloud code, it refers to `config.py` instead of the one I defined in `skygear/__init__.py`.

File: skygear/settings/module.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+class MissingModule(Exception):
+    pass
+
+
+loaded_modules = {}
+
+
+def _config_module(module, settings):
+    try:
+        module.includeme(settings)
```
Comment (rickmak) [inline skygear/settings/module.py]: Pass in the *args and **kwargs.

File: skygear/settings/module.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    except AttributeError as e:
+        msg = "Module {} does not implement function includeme"\
+            .format(module.__name__)
+        raise MissingImplementatation(msg)
```
Comment (rickmak) [inline skygear/settings/module.py]: Use the python build-in one? `NotImplementedError` https://docs.python.org/3/library/exceptions.html#NotImplementedError

File: skygear/settings/module.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+def add_module(name, module):
+    loaded_modules[name] = module
+
+
+def get_module(name):
+    try:
+        return loaded_modules[name]
+    except KeyError as ex:
+        msg = "Missing module {}".format(name)
```
Comment (rickmak) [inline skygear/settings/module.py]: Just throw `NameError: name 'module' is not defined`

**Comment by ben181231** [comment]
@rickmak Updated

**Pull request merged**